assets/ui-rework/hooks/supportScriptOutput.js

```diff
@@ -0,0 +1,74 @@
+import { Terminal } from "xterm"
+import { WebglAddon } from "xterm-addon-webgl"
+import { WebLinksAddon } from "xterm-addon-web-links"
+import { FitAddon } from "xterm-addon-fit"
+
+const defaultTermOptions = {
+  disableStdin: true,
+  cursorBlink: true,
+  cursorStyle: "bar",
+  macOptionIsMeta: true,
+  fontFamily: "Ubuntu Mono, courier-new, courier, monospace",
+  fontSize: 14,
+  theme: {
+    foreground: "#FFFAF4",
+    background: "#0E1019",
+    selectionBackground: "#48B9C7",
+    black: "#232323",
+    brightBlack: "#444444",
+    red: "#D82036",
+    brightRed: "#FF2740",
+    green: "#8CE10B",
+    brightGreen: "#ABE15B",
+    yellow: "#FFB900",
+    brightYellow: "#FFD242",
+    blue: "#007AD8",
+    brightBlue: "#0092FF",
+    magenta: "#6D43A6",
+    brightMagenta: "#9A5FEB",
+    cyan: "#00D8EB",
+    brightCyan: "#67FFF0",
+    white: "#FFFFFF",
+    brightWhite: "#FFFFFF"
+  }
+}
+
+const debounce = (func, time = 100) => {
+  let timer
+  return function(event) {
+    if (timer) clearTimeout(timer)
+    timer = setTimeout(func, time, event)
+  }
+}
+
+const resizeContent = (term, fitAddon) => {
+  term.resize(5, 5)
+  fitAddon.fit()
+}
+
+export default {
+  mounted() {
+    const term = new Terminal({ ...defaultTermOptions })
+
+    const fitAddon = new FitAddon()
+    term.loadAddon(fitAddon)
+    term.loadAddon(new WebglAddon())
+    term.loadAddon(new WebLinksAddon())
+
+    term.open(this.el)
+
+    let content = document.getElementById("support-script-output").textContent
+    console.log(content.split("\n"))
+    term.writeln(content.split("\n").join("\n\r"))
+
+    fitAddon.fit()
+
+    // resize terminal on window resize
+    window.addEventListener(
+      "resize",
+      debounce(() => {
+        resizeContent.apply(null, [term, fitAddon])
+      }, 300)
```

Comment:
```suggestion
      debounce(resizeContent.bind(null, term, fitAddon), 300)
```
No need to create an anonymous function and call `resizeContent`, this saves a call.

Reply:
removed this entirely and added some simplier row setting logic